test/benchmarks/material/checkbox/checkbox.perf-spec.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {$, browser} from 'protractor';
+import {runBenchmark} from '@angular/dev-infra-private/benchmark/driver-utilities';
+
+describe('checkbox overview performance benchmarks', () => {
+  beforeAll(() => {
+    browser.rootEl = '#root';
+  });
+
+  it('render (checked)', async() => {
+    await runBenchmark({
+      id: 'checkbox-overview-render-checked',
+      url: '',
+      ignoreBrowserSynchronization: true,
+      params: [],
+      setup: async () => {
+        await $('#show').click();
```

Comment:
I don't believe we use this `$` API anywhere else in our e2e tests, preferring `by.css()`. @crisbeto @mmalerba thoughts?

Reply:
Seems like `$(selector)` is just a shortcut for `element(by.css(selector))`

https://github.com/angular/protractor/blob/master/lib/element.ts#L1172
> Shortcut for querying the document directly with css.
> * `element(by.css('.abc'))` is equivalent to `$('.abc')`

Reply:
I don't mind it, it's a lot more concise.